docs/snippets/vue/table-story-fully-customize-controls.vue3.js.mdx

```diff
@@ -0,0 +1,28 @@
+```js
+// Table.stories.js
+
+const TableStory = (args) => ({
+  components: { Table },
+  setup() {
+    return { args };
+  },
+  template: `
+    <tr v-for="(row, idx1) in data">
+        <td v-for="(col, idx2) in row">
+            {{ data[idx1][idx2] }}
+        </td>
+    </tr>
+  `,
+});
+
+export const Numeric = TableStory.bind({});
+Numeric.args = {
+  //👇 This arg is for the story component
+  data: [
+    [1, 2, 3],
+    [4, 5, 6],
+  ],
+  //👇 The remaining args get passed to the `Table` component
+  size: 'large',
```

Comment:
I don't understand this? No table component is used in the template and `args` aren't bound in the template

Reply:
Let check the example i have. This one was always a bit tricky to convey with Vue

Reply:
With the latest commit i fixed the example. The Table component was included